sydent/db/invite_tokens.py

```diff
@@ -43,6 +56,19 @@ def getTokens(self, medium, address):
 
         for row in rows:
             medium, address, roomId, sender, token = row
+
+            # Ensure we're dealing with unicode.
+            if medium and isinstance(medium, bytes):
```

Comment:
```suggestion
            if isinstance(medium, bytes):
```

etc.   I assume that the empty string needs conversion, and anyway `isinstance` does the right thing with `None`